src/bindings/python/src/pyopenvino/core/common.cpp

```diff
@@ -194,18 +197,20 @@ void fill_tensor_from_strings(ov::Tensor& tensor, py::array& array) {
     if (tensor.get_size() != static_cast<size_t>(array.size())) {
         OPENVINO_THROW("Passed array must have the same size (number of elements) as the Tensor!");
     }
-    py::buffer_info buf = array.request();
+
+    const auto buf = array.request();
```

Comment:
```suggestion
    const py::buffer_info buf = array.request();
```

Reply:
Changed, but using auto in such cases give more robust code.